#goal definition
float64 push_distance
---
---
#feedback
